stm32mp1.mk

```diff
@@ -192,7 +192,8 @@ optee-os-clean: optee-os-clean-common
 ################################################################################
 # TrustedFirmware-A
 ################################################################################
-TFA_EXPORTS ?= CROSS_COMPILE="$(CCACHE)$(AARCH32_CROSS_COMPILE)"
+TFA_EXPORTS ?= CROSS_COMPILE="$(CCACHE)$(AARCH32_CROSS_COMPILE)" \
+	       CC="$(CCACHE)$(AARCH32_CROSS_COMPILE)gcc"
```

Comment:
You should probably also add a `LD="$(CCACHE)$(AARCH32_CROSS_COMPILE)ld"` like in https://github.com/OP-TEE/build/pull/793/.

Reply:
I saw that change but it seems TF-A does not need it. That said, maybe better in the long term, I'll add it. Thanks.

Reply:
Did you try building on an arm64 server? It does not seem to be needed when building on x86_64, although it may be because the default linker may be different on the two machines that I tested. Therefore setting LD in all cases is more likely to be correct.